book/src/concepts/event-channel.md

```diff
@@ -0,0 +1,120 @@
+# Event Channel
+
+This chapter will be easier than the previous ones. 
+
+While it is not essential to understand it to use amethyst, it can make your life much much easier in a lot of situations where using only data would make your code too complex.
+
+## What is an event channel?
+
+An `EventChannel` acts like a queue for any type that implements `Event`.
+The `Event` type is automatically implemented for all types that are `Send`, `Sync` and have a static lifetime.
+
+It is a single producer/multiple receiver. This means that it should be used when you only have a single "thing" (usually a system) producing events.
+In most case, the `EventChannel` should be stored in a global resource for ease of access. More on this later.
+
+
+The only exception to this is when you have a **SINGLE** external thread producing the events and you want your game to have a read access over the `EventChannel`.
+In this specific case, you can create a resource holding a Arc<Mutex<EventChannel<T>>>.
+
+If your use case involves multiple event producers, you should use some other type of event queuing method.
+In those case, you can use a `System` reading from a [mpsc channel](https://doc.rust-lang.org/std/sync/mpsc/) and moving the events into a `EventChannel`.
+You can also use other rust crates for that purpose.
+
+## Creating an event channel
+
+Super simple!
+
+```rust,ignore
+    // In the following examples, we are going to use i32 values as events.
+    let mut channel = EventChannel::<i32>::new();
```

Comment:
9 times out of 10 events should be some kind of enum.  I know this is just example code, but it might accidentally encourage bad practice.  Let's demonstrate this with a simple enum instead.